signer/src/storage/model.rs

```diff
@@ -339,6 +339,10 @@ pub struct EncryptedDkgShares {
     pub public_shares: Bytes,
     /// The set of public keys that were a party to the DKG.
     pub signer_set_public_keys: Vec<PublicKey>,
+    /// The threshold number of signature shares required to generate a
+    /// Schnorr signature.
```

Comment:
Can you add a little more detail for simpletons like me? Something like:

```
    /// The threshold number of signature shares required to generate a
    /// Schnorr signature, where a signature share is defined as  ...
```

Reply:
Done in https://github.com/stacks-network/sbtc/pull/694/commits/c3ceea7f8a43bcdd8ef975170e6c5d9aaf965461.